.ci/vgot.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [ $# = 0 ]; then
+    usage: vgot cmdpackage[@version]... >&2
+    exit 2
+fi
+d=`mktemp -d`
+cd "$d"
+echo 'module temp' > go.mod
```

Comment:
what's the advantage of this vs. `go mod init temp`?

Reply:
There is no real difference, if you prefer `go mod init temp` I can change it to this

Reply:
I think I would prefer it, yes.

Reply:
It should be faster.